content/docs/services/elasticsearch24.md

```diff
@@ -31,14 +31,79 @@ cf create-service elasticsearch24 1x my-elastic-service
 
 Each instance is one node; there is not currently a way to make multi-node instances.
 
-ElasticSearch 2.4 allows for dots in field names. This is a feature that existed
+Elasticsearch 2.4 allows for dots in field names. This is a feature that existed
 pre-ElasticSearch 2.0 but was disabled in versions 2.0 to 2.3. Learn more about
 that [here](https://www.elastic.co/guide/en/elasticsearch/reference/2.4/dots-in-names.html).
 
 ### Rotating credentials
 
 You can rotate credentials by creating a new instance and deleting the existing instance. If this is not an option, email [cloud.gov support](mailto:cloud-gov-support@gsa.gov) to request rotating the credentials manually.
 
+### Managing backups
+
+The Elasticsearch service includes the [AWS Cloud Plugin](https://www.elastic.co/guide/en/elasticsearch/plugins/2.4/cloud-aws.html), which supports snapshot and restore with AWS S3. For detailed instructions, see the [Snapshot and Restore](https://www.elastic.co/guide/en/elasticsearch/reference/2.4/modules-snapshots.html) [Cloud AWS Repository](https://www.elastic.co/guide/en/elasticsearch/plugins/2.2/cloud-aws-repository.html) documentation. You can also use this simple example. The example assumes you already have an elasticsearch service called `my-elasticsearch` and an app called `my-app`.
+
+* Create an instance of the [S3 service]({{< relref "docs/services/s3.md" >}}):
+
+    ```sh
+    cf create-service s3 basic my-s3-bucket
+    ```
+
+* Create a [service key](https://docs.cloudfoundry.org/devguide/services/service-keys.html) to access S3 credentials:
+
+    ```sh
+    cf create-service-key my-elasticsearch-backup my-key
+    ```
+
+* Connect to your elasticsearch service using port forwarding. Note: You'll need to leave the `cf ssh` command running and follow the next steps in a different terminal:
```

Comment:
I would mention we need to leave ssh running specifically for the localized port forwarding.